src/app/services/auth-service/auth.service.ts

```diff
@@ -7,30 +7,31 @@ import { Admin } from 'src/app/models/admin';
 @Injectable({
   	providedIn: 'root'
 })
+/**
+ * The authorization service.
+ */
 export class AuthService {
-	/**
-	 * This is the Authorization Service
-	 */
 
 
 	@Output() fireIsLoggedIn: EventEmitter<any> = new EventEmitter<any>();
 	 loggedIn: boolean = false;
 
 
 	/**
-	 * This is the constructor
-	 * @param router Creates a router instance
+	 * Creates a router instance
```

Comment:
This comment is slightly misleading. The constructor isn't creating a router instance. The router instance is being injected into the constructor by the framework. And the constructor is where we specify that we want a router instance.